### PR TITLE
Support 'RGBA' mode images by converting to 'RGB'

### LINKED ISSUE #12: PNG file support
Human-Pose-Estimator has [the same problem as Facial Age Estimator](https://github.com/IBM/MAX-Facial-Age-Estimator/issues/13). Image Caption Generator, Facial Recognizer, and Object Detector support both JPG and PNG files. On the other hand, Human-Pose-Estimator and Facial Age Estimator can handle only a JPG file. It will be easy for developers to switch MAX container which is related to image recognition if all containers support both JPG and PNG files.

### PR DESCRIPTION
**Pull request by MLnick**
Closes #12, #13

### CODE REVIEW

**Comment by MLnick** [comment]
cc @zuhito

**Comment by zuhito** [comment]
It looks great! After merging this, I will use PNG files using this Docker image.

File: core/model.py [hunk 1]
```diff
@@ -35,10 +37,16 @@ def __init__(self, path=DEFAULT_MODEL_PATH):
...
+        try:
+            image = Image.open(io.BytesIO(image_data))
+            if image.mode is not 'RGB':
+               image = image.convert('RGB')
+            # Convert RGB to BGR for OpenCV.
+            image = np.array(image)[:, :, ::-1]
+            return image
+        except IOError as e:
+            logger.error(str(e))
+            abort(400, "Please submit a valid image in PNG, TIFF or JPEG format")
```
Comment (ptitzler) [inline core/model.py]: The error message indicates that images in TIFF format are supported, yet there is no test case for it. I tried a TIFF image using the hosted instance (running the old code) and it raised HTTP error 500, just like the PNG image did.

**Comment by ptitzler** [comment]
Created a "test" image by converting one of our asset files from jpeg to tiff. File is ZIPped because GH can't handle TIFF.
[baby-bear.zip](https://github.com/IBM/MAX-Human-Pose-Estimator/files/2931803/baby-bear.zip)

**Comment by MLnick** [comment]
TIFF should be supported with this PR - I can add a test case.

Technically all formats loadable by Pillow are supported. There’s other
ones but JPG, PNG and TIFF are really the common ones.

**Comment by MLnick** [comment]
Added test case for `TIFF`, Travis is happy. Merging

**Pull request merged**